Soft Econ Adjustments/common/defines/zzz_MoG_SEA_def.txt

```diff
@@ -82,11 +82,13 @@ NEconomy = {
 	BUILDING_INITIAL_WAGE_WEEKS = 1 # 2			   					# A building will not adjust its wages for the first this many weeks of its existence
 	MAX_WAGE_STEP_CHANGE = 0.3 # 0.5 # 0.1			   					# Wage rates can change at most this much each week
 	
-	BUILDING_RAISE_WAGES_MIN_RELEVANT_POP_SIZE = 50					# Pops with less workforce than this are ignored when checking whether a profitable building should raise wages because of radicalizing workers
+	## For reference
+	# BUILDING_WAGES_MIN_RELEVANT_POP_SIZE = 50					# Pops with less workforce than this are ignored when checking whether a profitable building should raise wages because of radicalizing workers
```

Comment:
I've actually been tempted to lower this, as I think more granular consideration might help electricity and transport infrastructure to pull out of some hiring holes in low demand states - that might be more appropriate as an USU define though.

Reply:
Oh, I see this is now just gone